### ISSUE TITLE
dfx-generated private key PEM files don't parse with standard private key processing tools.

### ISSUE THREAD

**Issue by enzoh** [dfinity/sdk#1424]
Steps to reproduce:
```
openssl pkey -in ~/.config/dfx/identity/default/identity.pem 
```

```
unable to load key
140171254691136:error:0D0680A8:asn1 encoding routines:asn1_check_tlen:wrong tag:../crypto/asn1/tasn_dec.c:1149:
140171254691136:error:0D08303A:asn1 encoding routines:asn1_template_noexp_d2i:nested asn1 error:../crypto/asn1/tasn_dec.c:572:Field=attributes, Type=PKCS8_PRIV_KEY_INFO
140171254691136:error:0907B00D:PEM routines:PEM_read_bio_PrivateKey:ASN1 lib:../crypto/pem/pem_pkey.c:88:
```

**Comment by sesi200** [comment]
Internal ticket: [SDK-754](https://dfinity.atlassian.net/browse/SDK-754)

**Comment by jplevyak** [comment]
The quill generated identities do parse BTW.

**Comment by adamspofford-dfinity** [comment]
Fixed in #2741.
